frontend/src/js/reducers/Reducer.js

```diff
@@ -45,29 +135,48 @@ const projectReducer = (state = default_project, action) => {
 
 /*
 Tree Structure:
-
-projects:
     {
+        variables: [],
```

Comment:
My thought was an object with url and a label so we don't run into the naming issue we had in cdatgui.